src/controllers/recipe.controller.ts

```diff
@@ -87,6 +87,24 @@ export class RecipeController {
       return res.status(StatusCodes.INTERNAL_SERVER_ERROR).send();
     }
   }
+
+  public async readUserRecipesById(
+    req: Request,
+    res: Response
+  ): Promise<Response> {
+    try {
+      const { id: readUserId } = matchedData(req);
+
+      const recipes = await recipeService.readRecipeByUserId(readUserId);
+
+      if (!recipes) {
```

Comment:
You can do this a bit cleaner when you move constants, if statement and return statement outside trycatch block. That way later on you can easyly add another pice of code witout need to refactor.

Reply:
If I do it, then try{} block would be empty